test/CodeToRest_test.py

```diff
@@ -636,11 +649,605 @@ def test_35(self):
+
+    def test_71_c(self):
+        self.mt('DetailPrint "Hello World!"\n'
+                '; Comment here\n',
+                bf +
+                ' DetailPrint "Hello World!"\n' +
+                ef +
+                sl(-2) +
+                'Comment here\n', ['NSIS'])
+
+    def test_72_d(self):
+        self.mt('DetailPrint "Hello World!"\n'
+                '# Comment here\n',
+                bf +
+                ' DetailPrint "Hello World!"\n' +
+                ef +
+                sl(-2) +
+                'Comment here\n', ['NSIS'])
+
+    # TeX.
+    def test_72(self):
+        self.mt('Hello World!\n'
+                '% Comment here\n',
+                bf +
+                ' Hello World!\n' +
+                ef +
+                sl(-2) +
+                'Comment here\n', ['TeX'])
+
+    # Erlang.
+    def test_73(self):
+        self.mt('Hello World!.\n'
+                '% Comment here\n',
+                bf +
+                ' Hello World!.\n' +
+                ef +
+                sl(-2) +
+                'Comment here\n', ['Erlang'])
+
+    # QBasic.
+    def test_74(self):
+        self.mt('PRINT "Hello World!";\n'
+                '\' Comment here\n',
+                bf +
+                ' PRINT "Hello World!";\n' +
+                ef +
+                sl(-2) +
+                'Comment here\n', ['QBasic'])
+
+    # VB.net.
+    def test_75(self):
+        self.mt('Console.WriteLine("Hello World!")\n'
+                '\' Comment here\n',
+                bf +
+                ' Console.WriteLine("Hello World!")\n' +
+                ef +
+                sl(-2) +
+                'Comment here\n', ['VB.net'])
+
+    # REBOL.
+    def test_76(self):
+        self.mt('view [text "Hello World!"]\n'
+                '; Comment here\n',
+                bf +
+                ' view [text "Hello World!"]\n' +
+                ef +
+                sl(-2) +
+                'Comment here\n', ['REBOL'])
+
+    # LLVM.
+    def test_77_a(self):
+        self.mt('; Comment here\n',
+                sl(-3) +
+                'Comment here\n', ['LLVM'])
+
+    def test_77_b(self):
+        self.mt('Hello World!\n'
+                '; Comment here\n',
+                bf +
+                ' Hello World!\n' +
+                ef +
+                sl(-2) +
+                'Comment here\n', ['LLVM'])
+
+    # Ada.
+    def test_78(self):
+        self.mt('Put_Line("Hello World!");\n'
+                '-- Comment here\n',
+                bf +
+                ' Put_Line("Hello World!");\n' +
+                ef +
+                sl(-2) +
+                'Comment here\n', ['Ada'])
+
+    # Eiffel.
+    def test_79(self):
+        self.mt('print ("Hello World!%N")\n'
+                '-- Comment here\n',
+                bf +
+                ' print ("Hello World!%N")\n' +
+                ef +
+                sl(-2) +
+                'Comment here\n', ['Eiffel'])
+
+    # Vhdl.
+    """

```

Comment:
I tried both the uppercase and lowercase, and neither seemed to work. I'll triple check though and let you know.